packages/grafana-icons/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@grafana/saga-icons",
+  "version": "0.0.0",
+  "description": "Icons for Grafana",
+  "author": "Grafana Labs",
+  "license": "Apache-2.0",
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/grafana/grafana.git",
+    "directory": "packages/grafana-icons"
+  },
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "publishConfig": {
+    "main": "dist/index.js",
+    "module": "dist/esm/index.js",
+    "types": "dist/index.d.ts",
+    "access": "public"
+  },
+  "files": [
+    "./dist",
+    "./README.md",
+    "./CHANGELOG.md",
+    "./LICENSE_APACHE2"
+  ],
+  "scripts": {
+    "postinstall": "yarn generate",
```

Comment:
Just tested and it doesn't run if the install step fails. Since this also generates the index file, we need to make sure it's available as early as possible, otherwise the build will fail. Not sure of there's a better stage to generate icons?

Reply:
@jackw Could we do this with nx?

Reply:
It seems that excluding this package from workspaces either in package.json or lerna.json, also excludes it from the Nx project list, which makes it impossible to reference this project in the `dependsOn` scripts :(